EventBrite/Drivers/EventBriteFieldDisplayDriver.cs

```diff
@@ -0,0 +1,188 @@
+﻿using Etch.OrchardCore.Fields.EventBrite.Fields;
+using Etch.OrchardCore.Fields.EventBrite.Models;
+using Etch.OrchardCore.Fields.EventBrite.Models.Dto;
+using Etch.OrchardCore.Fields.EventBrite.Services;
+using Etch.OrchardCore.Fields.EventBrite.ViewModels;
+using Microsoft.Extensions.Localization;
+using Newtonsoft.Json;
+using OrchardCore.ContentManagement.Display.ContentDisplay;
+using OrchardCore.ContentManagement.Display.Models;
+using OrchardCore.DisplayManagement.ModelBinding;
+using OrchardCore.DisplayManagement.Views;
+using System;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace Etch.OrchardCore.Fields.EventBrite.Drivers
+{
+    public class EventBriteFieldDisplayDriver : ContentFieldDisplayDriver<EventBriteField>
+    {
+        #region PublicVariables
+
+        public IStringLocalizer T { get; set; }
+
+        #endregion PublicVariables
+
+        #region Dependencies
+
+        private IEventBriteSettingsService _eventBriteSettingsService;
+        private readonly IHttpClientFactory _clientFactory;
+
+        #endregion Dependencies
+
+        #region Constructor
+
+        public EventBriteFieldDisplayDriver(IStringLocalizer<EventBriteFieldDisplayDriver> localizer, IEventBriteSettingsService eventBriteSettingsService, IHttpClientFactory clientFactory)
+        {
+            T = localizer;
+            _eventBriteSettingsService = eventBriteSettingsService;
+            _clientFactory = clientFactory;
+        }
+
+        #endregion Constructor
+
+        #region ContentFieldDisplayDriver<EventBriteField>
```

Comment:
Spoken to @lawrencedine about this on another PR for some client work. Change this to "Implementation". If the class implements multiple interfaces then use sub regions for each interface.